test.rb

```diff
@@ -1,11 +1,20 @@
 require './person'
 require './capitalize_Decorator'
 require './trimmer_decorator'
+require './classroom'
+require './student'
+require './book'
+require './rental'
 
-person = Person.new(22, 'maximilianus')
-capitalize_person = CapitalizeDecorator.new(person)
-trim_person = TrimmerDecorator.new(capitalize_person)
+math = Classroom.new('Algebra')
```

Comment:
I like the way you approach testing the project! 
However, kindly implement the suggestions I gave.